templates/_base.html

```diff
@@ -22,7 +23,16 @@
         style="display: none;">
 
         {% for child in link.children %}
-        <a href="{{ child.href }}" class="header_button">{{ child.label }}</a>
+            {% if child.action_url %}
+            <form method="post" action={{ child.action_url}}>
+                    {% csrf_token %}
+                    {{ form | crispy }}
```

Comment:
This is going to cause bugs.

Any time a template is rendered with with a context including a `form` then then the form will be rendered in the heading.

Eg:


![image](https://github.com/user-attachments/assets/5e6afae1-5f98-47a8-94df-50ea8891e67d)

```
from django import forms


class MyForm(forms.Form):
    name = forms.CharField(max_length=100, required=True, label="Your Name")


def page_home(request):
    context = {"form": MyForm()}
    return render(request, "page_home.html", context=context)
```

Of course this is a silly example, but we use forms in the cfp and review process. 

It looks like all you want here is a logout button that works. The logout form doesnt have any visable fields. So you could leave out the {{ form | crispy}} line entirely and then that bug will go away. 

That said, there is another downside to this approach in that we'll need to make sure this works responsively. We are going to need to make the header area responsive at some point.

There is a much simpler approach. 

Add this to the base settings:
```
ACCOUNT_LOGOUT_ON_GET = True
```

Then a normal get request to logout will log the user out. So then all the action_url functionality could be removed. 

There are downsides to letting people log out using a GET request, but I dont think we are in any danger. 

So to summarise,

I think it would be good to just use a GET to logout. But if you really want to keep the POST then it's important to remove the `{{form | crispy}}` line

Reply:
fixed

Reply:
This isn't fixed. It still has `{{ form | crispy }}` in the header area. So if any view has a contect that includes something named `form` then it will show up in the header area. 

This will be bad because we use a lot of forms. So a lot of pages will end up with a bug.

Please remove the form from the header area. 

Take a look at the comment I left before. If you read from the top then you'll see a demonstration of the bug.

Reply:
If the action_url stuff isn't being used then it would be good to remove it. Then it wont be able to cause any bugs.

Reply:
i've forgot to clean it. now it's ok